### PR TITLE
[Durable Objects] Alarm restructuring

### PR DESCRIPTION
**Pull request by joshthoward**

### CODE REVIEW

**Review by irvinebroque — APPROVED** [review]
🙌

File: content/durable-objects/api/alarms.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+  - Unset the alarm if there is a currently set alarm.
+
+  - Calling `deleteAlarm()` inside the `alarm()` handler may prevent retries on a best-effort basis, but is not guaranteed.
+
+{{</definitions>}}
+
+---
+
+## Handler Methods
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/api/alarms.md]:
```suggestion
## Handler methods
```

File: content/durable-objects/api/alarms.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Alarms are more fine grained than [Cron Triggers](/workers/configuration/cron-triggers/#cron-triggers). A Worker can have up to three Cron Triggers configured at once, but it can have an unlimited amount of Durable Objects, each of which can have an alarm set.
+
+Alarms are directly scheduled from within your Durable Object. Cron Triggers, on the other hand, are not programmatic. [Cron Triggers](/workers/configuration/cron-triggers/#cron-triggers) execute based on their schedules, which have to be configured through the Cloudflare dashboard or API.
+
+{{</Aside>}}
+
+Alarms can be used to build distributed primitives, like queues or batching of work atop Durable Objects. Alarms also provide a mechanism to guarantee that operations within a Durable Object will complete without relying on incoming requests to keep the Durable Object alive. For a complete example, see [Using the Alarms API](/durable-objects/examples/alarms-api/).
+
+## Transactional Storage Methods
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/api/alarms.md]:
```suggestion
## Transactional Storage methods
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Review by Maddy-Cloudflare — APPROVED** [review]

**Review by kodster28 — APPROVED** [review]
If it breaks, we can always revert

**Pull request merged**